.github/workflows/release.yaml

```
@@ -1,39 +1,117 @@
-name: release
+# This code is borrowed from https://github.com/kubernetes-sigs/cluster-api/blob/main/.github/workflows/release.yaml
+name: Create Release
 
 on:
   push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-    - "v*"
+    branches:
+      - main
```

Indents in the file are wrong, please fix.

```suggestion
    - main
```

---

seems fixed now